app/views/docs/functions-deploy.phtml

```diff
@@ -0,0 +1,216 @@
+<p>
+    Appwrite Functions are mini-applications in Appwrite with their own endpoints. 
+    Each function can have many deployments, which can be thought of as versions of the mini-application.
+</p>
+
+<p>
+    Functions can be created and deployed in different ways to meet your unique development habits.
+    You can automatically deploy Appwrite Functions from source control, build your own deployment pipelines using the Appwrite CLI, or upload code files manually.
+    Here's everything you need to know to deploy your first Appwrite Function.
+</p>
+
+<h2><a href="#vcs" id="vcs">VCS (Version Control System)</a></h2>
```

Comment:
I am not entirely onboard with calling it VCS in docs. I think calling it Git, and mentioning GitHub would feel more natural to developer. You can double check with Eldad.